src/main/java/mate/academy/hibernate/relations/model/Movie.java

```diff
@@ -1,11 +1,25 @@
 package mate.academy.hibernate.relations.model;
 
+import jakarta.persistence.Entity;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.JoinTable;
+import jakarta.persistence.ManyToMany;
 import java.util.ArrayList;
 import java.util.List;
 
+@Entity
 public class Movie implements Cloneable {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
     private String title;
+    @ManyToMany(fetch = FetchType.LAZY)
+    @JoinTable(name = "movie_actor", joinColumns = @JoinColumn(name = "movie_id"),
+                    inverseJoinColumns = @JoinColumn(name = "actor_id"))
     private List<Actor> actors;
```

Comment:
```suggestion
    @JoinTable(name = "movie_actor", joinColumns = @JoinColumn(name = "movie_id"),
                    inverseJoinColumns = @JoinColumn(name = "actor_id"))
    private List<Actor> actors;
```

many-to-many and one-to-many are LAZY by default

Reply:
it`s not gonna working